frontend/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@apollo/client": "^3.3.16",
     "@chakra-ui/icon": "^1.1.11",
+    "@chakra-ui/icons": "^1.0.15",
```

Comment:
We currently use MdIcons + charkra-ui/icon for icons. An example of using MdIcons & Icons (from chakraui) together can be found [here](https://github.com/uwblueprint/planet-read/blob/c7ff9bdec70b8b91ce2c34f8c2085c8fff7c57d3/frontend/src/components/pages/HomePage.tsx#L107). We ought to stick to using MdIcon for design consistency & reducing the number of dependencies in our frontend; plz remove this package and use MdIcons & Icons (from chakraui).

[This website ](https://react-icons.github.io/react-icons/icons?name=md)will help you find the MdIcon equivalents to the icons you're currently using in this pr.